main.tf

```diff
@@ -5,15 +5,19 @@
  *
  * * acm-certificate-expiration-check: Ensure ACM Certificates in your account are marked for expiration within the specified number of days.
  * * cloudtrail-enabled: Ensure CloudTrail is enabled.
+ * * ec2-volume-inuse-check: Checks whether EBS volumes are attached to EC2 instances
+ * * guardduty-enabled-centralized: Checks whether Amazon GuardDuty is enabled in your AWS account and region.
  * * iam-password-policy: Ensure the account password policy for IAM users meets the specified requirements.
  * * iam-user-no-policies-check: Ensure that none of your IAM users have policies attached; IAM users must inherit permissions from IAM groups or roles.
  * * instances-in-vpc: Ensure all EC2 instances run in a VPC.
  * * root-account-mfa-enabled: Ensure root AWS account has MFA enabled.
+ * * rds-storage-encrypted: Checks whether storage encryption is enabled for your RDS DB instances.
+ * * s3-bucket-public-write-prohibited: Checks that your S3 buckets do not allow public write access.
```

Comment:
There are going to be AWS accounts that want some number of public S3 buckets so at the very least we should be able to disable this compliance check. If we could feed in a list of exempt S3 buckets which are expected to be accessible publicly, that would be ideal.

Reply:
This S3 rule is only for public write and doesn't validate read access.  I'm pretty sure you never want public write even if a bucket is public read, right?

Reply:
Ah, I missed that it's only checking for write access and not read. Nevermind!